hotshot-query-service/src/data_source/storage.rs

```diff
@@ -214,6 +216,10 @@ where
         common: VidCommonQueryData<Types>,
         share: Option<VidShare>,
     ) -> impl Send + Future<Output = anyhow::Result<()>>;
+    fn insert_state_cert(
```

Comment:
Where does this get called?

Reply:
called here: https://github.com/EspressoSystems/espresso-network/pull/2917/files#diff-d6d79e95846b7f888aec7a5acbaa9e53645539fbf0334e30bc674c8317b7f6f2R118